### ISSUE TITLE
TZ3000_TS0043_3_gang_scene_switch

### ISSUE THREAD

**Issue by Dewcal** [dresden-elektronik/deconz-rest-plugin#7114]
## Device

- Product name: The device name as shown on the product or package.
- Manufacturer: As per deCONZ GUI Basic cluster.
- Model identifier: As per deCONZ GUI Basic cluster.
- Device type : Please remove all unrelated device types. 
  - Light
  - Lock
  - Remote
  - Sensor
  - Siren
  - Switch
  - Thermostat
  - Other:

**Comment by Mimiix** [comment]
The template is empty. Please check the Forum post where i provided all links with info on how to do.

**Comment by Dewcal** [comment]
I suspect I am not requesting help in the correct place as this is new to me!  I have just obtained a 3 gang scene switch as in this link https://www.aliexpress.com/item/1005005555852288.html?spm=a2g0o.order_list.order_list_main.16.741c1802oRjFhI which is, I believe, based on TUYA.
While it appears on the deCONZ GUI as below it doe snot fully integrate nor appear in Homeseer 4
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/67413046/0b4c86b5-d7d4-4864-be58-9e40f9d50db8)
the basic cluster info is totally blank whilst the node info 
![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/67413046/16eedae0-1008-4717-b8fe-da69b1a158dd)
is not very helpful.
I have tried to "pair" the unit several times to no change.
Any help to get this working would be appreciated.
